### ISSUE TITLE
About the number of images

### ISSUE THREAD

**Issue by seiyaito** [sunghoonim/DPSNet#7]
Hi,

Thank you for sharing the code.

I would like to ask you how to produce Figure 7 in your paper.
I think that test sets in MVS, SUN3D, RGBD, and Scenes11 contain only two views, but your evaluation has done using more than two views.
Which dataset do you use?

**Comment by sunghoonim** [comment]
For the ablation study, I use the validation set, which randomly picked before training.

**Comment by seiyaito** [comment]
Thank you for your prompt replay.

I used your pretrained model and got similar results on the validation set.
I also tested the MVS training set, but the accuracy decreases as the number of images increases.
Although the MVS dataset was not used for training, the results on the MVS test set (two views) seem good.
Do you have any idea of what could be happening?

**Comment by naveenventuri** [comment]
Hi seiyaito,
I am not able to match the results with the pretrained weights on the validation sets could you tell the configuration on which you got the matching result

**Comment by seiyaito** [comment]
Hi @naveenventuri ,

I used the pretrained model ([pretrained/dpsnet.pth.tar](https://github.com/sunghoonim/DPSNet/blob/master/pretrained/dpsnet.pth.tar])) and the split list ([dataset/train/val.txt](https://github.com/sunghoonim/DPSNet/blob/master/dataset/train/val.txt)).
I just run test.py with different sequence lengths.

**Comment by naveenventuri** [comment]
Hi @seiyaito ,
Thanks for the reply
I tried running with the same model and list but still i am not able to replicate the metrics given on the paper . Is there any effect of sequence length on the output ? If so may i know at what sequence length are you able to replicate the results
Thanks

**Comment by seiyaito** [comment]
Hi, @naveenventuri ,

I might have made a mistake so please let me check.
What are you going to replicate?
If you want to replicate the metrics in Table 1, I couldn't replicate it.

I asked how to reproduce Fig. 7 in this issue #7.
I'm not sure that I definitely replicated the metrics on the paper because the effect of the number of images is shown in the graph.
But, the similar results have been observed: as the number of images increases, the error decreases.
